### PR TITLE
Add copy operator to wxChartLabel 

### PR DESCRIPTION
**Pull request by pcallycat**
getting compile error on macOS due to 'implicitly deleted copy operator'.  defined copy operator to make library compile.

### CODE REVIEW

File: include/wx/charts/wxchartlabel.h [hunk 1]
```diff
@@ -55,6 +55,8 @@ class wxChartLabel : public wxChartElement
     wxChartLabel(const wxString &text, wxDouble width,
         wxDouble height, const wxChartLabelOptions &options);
 
+    wxChartLabel &operator=(const wxChartLabel &obj);
```
Comment (xleclercq) [inline include/wx/charts/wxchartlabel.h]: If we are adding an assignment operator we should also add a copy constructor.

File: src/wxchartlabel.cpp [hunk 2]
```diff
@@ -40,6 +40,15 @@ wxChartLabel::wxChartLabel(const wxString &text,
 {
 }
 
+wxChartLabel &wxChartLabel::operator=(const wxChartLabel &obj) {
+    this->m_options=obj.m_options;
```
Comment (xleclercq) [inline src/wxchartlabel.cpp]: Please remove the "this->". We already use the "m_" prefix to indicate member variables.

File: include/wx/charts/wxchartlabel.h (hunk 1 above)
Comment (pcallycat) [reply include/wx/charts/wxchartlabel.h]: I've added the copy constructor to the header file

File: src/wxchartlabel.cpp (hunk 2 above)
Comment (pcallycat) [reply src/wxchartlabel.cpp]: removed "this->" and implemented the copy constructor

**Pull request closed without merging**